.github/CONTRIBUTING.md

```diff
@@ -25,7 +26,7 @@
 
 Make sure you have [Node.js][1] version v8 or later installed.
 
-You can contribute to Stardust by being an official [contributor](setup-local-development.md#contributors) or without permissions, as a [collaborator](setup-local-development.md#collaborators)
+You can contribute to FluentUI by being an official [contributor](setup-local-development.md#contributors) or without permissions, as a [collaborator](setup-local-development.md#collaborators)
```

Comment:
Curious of the reasoning for using FluentUI rather than Fluent UI? I know that more directly matches the scope name, but I thought the lack of a dash in the scope name was just an unfortunate artifact of fluent-ui being taken already.

Reply:
Per Joseph McLaughlin, they got a lot of feedback on the `FluentUi` initial idea and they changed the official guidance.  It is now officially `Fluent UI`. 👍

He said he'd make an official announcement to the internal team today.  Not sure if it happened yet.  I do have the design guidance PDF if you'd like a copy 😄